src/providers/historical-telemetry-provider.js

```diff
@@ -145,7 +145,7 @@ export default class YamcsHistoricalTelemetryProvider {
     isImagery(domainObject) {
         let metadata = this.openmct.telemetry.getMetadata(domainObject);
 
-        return metadata.valuesForHints(['image']).length !== 0;
+        return metadata && metadata.valuesForHints(['image']).length !== 0;
```

Comment:
Was there an issue with metadata not being defined? It should always be defined.

Reply:
I think it was when I was trying to use old providers for fault and fault mgmt does not have metadata